<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_AIAssit</class>
 <widget class="QWidget" name="Frm_AIAssit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GKG AI助手</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Frm_AIAssit {
    background-color: #f0f2f5;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
    color: #333;}</string>
  </property>
  <layout class="QHBoxLayout" name="mainHorizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="ChatList" name="ChatListWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnNewConversation {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #667eea, stop:1 #764ba2);
    color: white;
    border: none;
    border-radius: 12px;
    font-weight: 600;
    font-size: 14px;
    padding: 12px 20px;
    margin: 8px;
    min-height: 44px;
    text-align: left;
}

QPushButton#btnNewConversation:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #7c8ef0, stop:1 #8a5cae);
    transform: translateY(-2px);
}

QPushButton#btnNewConversation:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #5a6fd8, stop:1 #6b4596);
}

/* =================== 对话列表样式 =================== */
QListWidget#m_conversationList {
    border: none;
    background: transparent;
    outline: 0;
    padding: 8px;
}

/* =================== 对话项基础样式 =================== */
QListWidget#m_conversationList::item {
    padding: 16px 20px;
    margin: 4px 8px;
    border-radius: 12px;
    color: #555555;
    font-weight: 500;
    font-size: 14px;
    background: rgba(255, 255, 255, 0.6);
    border: 1px solid rgba(255, 255, 255, 0.3);
    min-height: 20px;
    max-width:50px;
}

/* =================== 对话项悬停效果 =================== */
QListWidget#m_conversationList::item:hover {
    background: rgba(102, 126, 234, 0.15);
    color: #555555;
    border: 1px solid rgba(102, 126, 234, 0.3);
    transform: translateX(4px);
}

/* =================== 对话项选中状态 =================== */
QListWidget#m_conversationList::item:selected {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 rgba(102, 126, 234, 0.25), stop:1 rgba(118, 75, 162, 0.25));
    color: #667eea;
    font-weight: 700;
    border: 2px solid rgba(102, 126, 234, 0.5);
}

/* =================== 对话项选中且悬停 =================== */
QListWidget#m_conversationList::item:selected:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 rgba(102, 126, 234, 0.35), stop:1 rgba(118, 75, 162, 0.35));
    color: #667eea;
    font-weight: 700;
    border: 2px solid rgba(102, 126, 234, 0.7);
}

/* =================== 滚动条样式 =================== */
QListWidget#m_conversationList QScrollBar:vertical {
    background: rgba(0, 0, 0, 0.05);
    width: 6px;
    border-radius: 3px;
    margin: 2px;
}

QListWidget#m_conversationList QScrollBar::handle:vertical {
    background: rgba(102, 126, 234, 0.3);
    border-radius: 3px;
    min-height: 20px;
}

QListWidget#m_conversationList QScrollBar::handle:vertical:hover {
    background: rgba(102, 126, 234, 0.5);
}

QListWidget#m_conversationList QScrollBar::add-line:vertical,
QListWidget#m_conversationList QScrollBar::sub-line:vertical {
    height: 0;
    width: 0;
}</string>
     </property>
     <layout class="QVBoxLayout" name="conversationLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ChatShowWidget" name="ChatShow" native="true">
        <property name="styleSheet">
         <string notr="true">/* =================== 头部区域样式 =================== */
QWidget#chatHeader {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
        stop:0 #ffffff, stop:1 #f8f9fb);
    border: none;
    border-radius: 6px;
    border-bottom: 1px solid #ebeef5;
}

/* =================== 标题样式 =================== */
QLabel#chatTitle {
    color: #2c3e50;
    font-weight: bold;
    padding: 0 15px;
}

/* =================== 切换按钮样式 =================== */
QPushButton#toggleButton {
    background-color: transparent;
    border: none;
    border-radius: 6px;
    font-size: 24px;
    padding: 8px;
    min-width: 40px;
    min-height: 40px;
    max-width: 40px;
    max-height: 40px;
}

QPushButton#toggleButton:hover {
    background-color: rgba(102, 126, 234, 0.1);
}

QPushButton#toggleButton:pressed {
    background-color: rgba(102, 126, 234, 0.2);
}

QPushButton#toggleButton:focus {
    outline: none;
}

/* =================== 参数设置按钮样式 =================== */
QPushButton#btnParamSet {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #e6a23c, stop:1 #d4941a);
    color: white;
    border: none;
    border-radius: 6px;
    font-weight: 600;
    font-size: 13px;
    padding: 8px 16px;
}

QPushButton#btnParamSet:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #ebb563, stop:1 #e6a23c);
}

QPushButton#btnParamSet:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
        stop:0 #d4941a, stop:1 #c7871a);
}

/* =================== 聊天框架样式 =================== */
QFrame#frame {
    background: white;
    border-radius: 10px;
    border: 1px solid #ebeef5;
}

/* =================== 聊天列表样式 =================== */
QListWidget#listWgChatFrame {
    border: none;
    background: transparent;
    outline: 0;
}

/* =================== 空状态样式 =================== */
QWidget#emptyStateWidget {
    background: transparent;
    border: none;
}

QLabel#emptyIconLabel {
    background: transparent;
    border: none;
}

QLabel#emptyTextLabel {
    color: #606266;
    background: transparent;
    border: none;
    font-size: 18px;
    font-weight: 500;
    line-height: 1.4;
}

/* =================== 滚动条样式 =================== */
QListWidget#listWgChatFrame QScrollBar:vertical {
    background: transparent;
    width: 8px;
    margin: 0px 0px 0px 0px;
    border-radius: 4px;
}

QListWidget#listWgChatFrame QScrollBar::handle:vertical {
    background: #c0c4cc;
    min-height: 20px;
    border-radius: 4px;
}

QListWidget#listWgChatFrame QScrollBar::handle:vertical:hover {
    background: #909399;
}

QListWidget#listWgChatFrame QScrollBar::add-line:vertical,
QListWidget#listWgChatFrame QScrollBar::sub-line:vertical {
    height: 0;
    width: 0;
}

/* =================== 列表项样式 =================== */
QListWidget#listWgChatFrame::item {
    margin: 4px 0;
    background: transparent;
}

QPushButton#UpButton, QPushButton#DownButton {
    background-color: transparent;
    border: none;
    border-radius: 15px;
    margin: 2px;
}

QPushButton#UpButton:hover, QPushButton#DownButton:hover {
    background-color: rgba(102, 126, 234, 0.1);
}

QPushButton#UpButton:pressed, QPushButton#DownButton:pressed {
    background-color: rgba(102, 126, 234, 0.2);
}

QPushButton#UpButton:focus, QPushButton#DownButton:focus {
    outline: none;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChatInputWidget" name="ChatInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="AIParamWidget" name="AIParams" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 8px;
    padding: 12px;
}
QLabel {
    font-weight: 600;
    color: #606266;
}
QLineEdit, QComboBox, QSpinBox, QDoubleSpinBox {
    border: 1px solid #c0c4cc;
    border-radius: 6px;
    padding: 5px 8px;
    background: #fff;
    selection-background-color: #409eff;
    selection-color: white;
}
QCheckBox {
    spacing: 8px;
    color: #606266;
    font-weight: 500;
}
QPushButton {
    background-color: #409eff;
    border: none;
    color: white;
    padding: 8px 16px;
    border-radius: 6px;
    font-weight: 600;
    min-width: 80px;
    min-height: 28px;
}
QPushButton:hover {
    background-color: #66b1ff;
}
QPushButton:pressed {
    background-color: #3a8ee6;
}</string>
     </property>
     <layout class="QVBoxLayout" name="parameterLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChatInputWidget</class>
   <extends>QWidget</extends>
   <header location="global">AIAssit\chatinputwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatShowWidget</class>
   <extends>QWidget</extends>
   <header location="global">AIAssit\chatshowwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AIParamWidget</class>
   <extends>QWidget</extends>
   <header location="global">AIAssit\aiparamwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatList</class>
   <extends>QWidget</extends>
   <header location="global">AIAssit\ChatList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
